content/docs/5.1/deploy/operate/scale.md

```diff
@@ -146,12 +146,12 @@ You can take similar steps to remove the PD node.
 
     The `--node` parameter is the ID of the node to be taken offline.
 
-    If you see the `Scaled cluster <cluster-name> in successfully`, the scale-in operation is successfully completed.
-
+    If you see the message "Scaled cluster in successfully", it means that the scale-in operation is successfully completed.
+    
+    Besides, since the scale-in process takes some time, the status of the node to be taken offline became `Tombstone` also indicates that the scale-in operation is successfully completed.
```

Comment:
```suggestion
    Besides, since the scale-in process takes some time, the status of the node to be taken offline became `Tombstone` also indicates that the scale-in operation is successfully completed.
```
为什么要用 since 呢？这句话有因果关系吗？没有读出来呢。另外，后半句 ‘’became " 和 “indicates ”，两个谓语，感觉有些读不通呢

Reply:
这句的中文应该是想表达：另外，由于缩容过程需要花费一些时间，因此，需要删除的节点的状态变成 “Tombstone” 也能表示缩容成功。

 改成下面这样，把句子拆分为两句，是否能表达地更加清晰一些呢？ @Liuxiaozhen12 

```suggestion
    Besides, if the status of the node to be taken offline became `Tombstone`, it also indicates that the scale-in operation is successfully completed because the scale-in process takes some time.
```